src/main/webapp/resources/js/pages/cart/components/Cart.jsx

```diff
@@ -1,23 +1,23 @@
-import React, { lazy, Suspense, useEffect, useState } from "react";
-import { connect } from "react-redux";
-import PropTypes from "prop-types";
 import { Layout } from "antd";
-import { actions } from "../../../redux/reducers/cart";
+import React, { lazy, Suspense, useState } from "react";
```

Comment:
Remove `useState` and use React.useState below instead

Reply:
Good catch, updated in 4bf7933335a355a16f1c1a4f4eb34b66014b2d76